tutorials/example_keras_imagenet.ipynb

```diff
@@ -0,0 +1,379 @@
+   "source": [
+    "# Quantization using the Model Compression Toolkit - ImageNet dataset example"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "930e6d6d-4980-4d66-beed-9ff5a494acf9",
+   "metadata": {},
+   "source": [
+    "## Overview"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "699be4fd-d382-4eec-9d3f-e2e85cfb1762",
+   "metadata": {},
+   "source": [
+    "This quick start guide covers how to use the Model Compression Toolkit (MCT) for quantizing a pre-trained model on ImageNet. We will do so by giving an example, quantizing a pre-trained model, then evaluating the accuracy on ImageNet dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "85199e25-c587-41b1-aaf5-e1d23ce97ca1",
+   "metadata": {},
+   "source": [
+    "## Summary"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9c0e9543-d356-412f-acf1-c2ecad553e06",
+   "metadata": {},
+   "source": [
+    "In this tutorial we will cover:\n",
+    "\n",
+    "1. Loading and preprocessing ImageNet validation dataset\n",
+    "2. Loading and preprocessing ImageNet unlabeled representative dataset\n",
+    "3. Post-Training-Quantization using MCT\n",
+    "4. Accuracy evaluation of the floating-point and the quantized models"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "04228b7c-00f1-4ded-bead-722e2a4e89a0",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2657cf1a-654d-45a6-b877-8bf42fc26d0d",
+   "metadata": {},
+   "source": [
+    "Install the relevant packages:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "324685b9-5dcc-4d22-80f4-dec9a93d3324",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!pip install -q model-compression-toolkit\n",
+    "!pip install -q tensorflow\n",
+    "!pip install -q tensorflow-model-optimization"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "b3f0acc8-281c-4bca-b0b9-3d7677105f19",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import tensorflow as tf\n",
+    "import keras\n",
+    "import model_compression_toolkit as mct\n",
+    "from keras.applications.mobilenet_v2 import MobileNetV2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0c7fed0d-cfc8-41ee-adf1-22a98110397b",
+   "metadata": {},
+   "source": [
+    "## Dataset preparation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "719a8ca4-6c62-4624-a1ec-662c03dde902",
+   "metadata": {},
+   "source": [
+    "Define dataset folders:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "c9893131-0a95-4472-aa42-a73bd8d50576",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "TEST_DATASET_FOLDER = '/data/projects/swat/datasets_src/ImageNet/ILSVRC2012_img_val_TFrecords'\n",
+    "TRAIN_DATASET_FOLDER = '/data/projects/swat/datasets_src/ImageNet/ILSVRC2012_img_train'"
```

Comment:
Please change these paths to something general. Something like "/path/to/imagenet/training/dir" and so on...